bundles/org.openhab.binding.sonos/src/test/java/org/openhab/binding/sonos/internal/SonosXMLParserTest.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.sonos.internal;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.junit.jupiter.api.Test;
+
+/**
+ *
+ * @author Laurent Garnier - Initial contribution
+ */
+@NonNullByDefault
+public class SonosXMLParserTest {
+
+    @Test
+    public void extractModelName() {
+        assertEquals("Move", SonosXMLParser.extractModelName("Sonos Move"));
+        assertEquals("Roam SL", SonosXMLParser.extractModelName("Sonos Roam SL"));
+        assertEquals("Five", SonosXMLParser.extractModelName("Sonos Five (OpenHome)"));
+        assertEquals("One SL", SonosXMLParser.extractModelName("Sonos One SL (OpenHome)"));
```

Comment:
Now getting really picky/detailed, but consider if you want also to add a case for the removing of ":", so test coverage would be complete. :-) I'm ready to merge as-is since you already provided coverage for your changes and then some, so in much better condition now.

Reply:
I forget the case of PLAY:5.
I am adding it just now...

Reply:
Done

Reply:
Perfect!